src/components/Modals/AddContactModal.jsx

```diff
@@ -62,6 +75,28 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
   const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
   const [oidcProviders] = useState([...ENV.VITE_SUGGESTED_OIDC_OPTIONS.split(', '), 'Other']);
   const [Oidc, setOIDC] = useState('');
+  const [deleteViaEdit, setDeleteViaEdit] = useState(false);
+
+  // Planning and observations
+  // try to add new contact to pod A, which already has a contact
+  // -result: updates existing contact
+  // add new contact to pod B, which doesn't have a contact
+  //   -result: adds to grid
+  // current commit : Edit function will update contact names if webId remains the same.
+  // If user edits the webId, a new contact will be created.
+
+  // ✅ prepopulate form when editing.
+  // ✅ if webId is different, delete current contact and add contact. Having issues with re-render
+  // ✅ warning if webId already exists
```

Comment:
I don't think we need the whole implementation detail in here, maybe a one line comment for where the functionality is